stack-lts-22.yaml

```diff
@@ -0,0 +1,38 @@
+resolver: lts-22.40
```

Comment:
Would prefer this to be the latest iteration of `lts-22` (i.e. 22.43) but it's not that big of a deal. We mostly use the LTS-es for making sure it builds with different GHC versions.

Reply:
I was also considering updating this. I've updated it.